content/4_Gold/Modular.mdx

```diff
@@ -328,7 +328,7 @@ int main() {
 Because it takes $\mathcal{O}(\log p)$ time to compute a modular inverse modulo
 $p$, frequent use of division inside a loop can significantly increase the
 running time of a program. If the modular inverse of the same number(s) is/are
-being used many times, it is a good idea to precalculate it.
+being used many times, it is a good idea to precalculate it. Note the above problem could also be solved by precalculating invererses by first calculating the inverse of n! for a large n then finding all previous values by multiplying in $O(n)$.
```

Comment:
would like to see this in the code. 

+ typos

+ n! and n should be surrounded by $$

Reply:
as a comment?

Reply:
actual code

Reply:
add \mathcal too